_posts/en/2024-10-09-wallet-integration-guide.md

```diff
@@ -0,0 +1,590 @@
+
+# Replaced prior package
+bitcoin-cli -regtest getrawmempool
+[
+  "fdee3b6a5354f31ce32242db10eb9ee66017e939ea87db0c39332262a41a424b"
+]
+
+# For child, burn value to fees, make 65vbyte tx with OP_RETURN to avoid tx-size-small error
+bitcoin-cli -regtest createrawtransaction '[{"txid": "fdee3b6a5354f31ce32242db10eb9ee66017e939ea87db0c39332262a41a424b", "vout": 1}]' '[{"data": "feeeee"}]'
+02000000014b421aa4622233390cdb87ea39e91760e69eeb10db4222e31cf354536a3beefd0100000000fdffffff010000000000000000056a03feeeee00000000
+
+# No signing required; it’s witness-free segwit
+bitcoin-cli -regtest -rpcwallet=test sendrawtransaction 02000000014b421aa4622233390cdb87ea39e91760e69eeb10db4222e31cf354536a3beefd0100000000fdffffff010000000000000000056a03feeeee00000000
+8d092b61ef3c1a58c24915671b91fbc6a89962912264afabc071a4dbfd1a484e
+
+```
+
+## User Stories
+
+Moving on from the more general release notes level feature description, we will
+describe a few common wallet patterns and how these can benefit from these
+updates, with or without the wallets making active changes.
+
+### Simple Payments
+
+If users wish to have more predictable RBF behavior, one way would be to opt-in
+to TRUC transactions. If adapted, wallets should only use confirmed outputs for
+TRUC transactions and construct them to stay within 10kvB (as opposed to the
+100kvB non-TRUC limit). This restricted limit still supports larger batch
+payments as well. If a wallet has no choice but to spend an unconfirmed input,
+if that input comes from a TRUC transaction, this new transaction would need to
+be below 1kvB.
+
+This would allow an RBFing wallet confidence that receivers of bitcoin would not
+be able to create arbitrary chains of transactions off of the payment, pinning
+the user. Incoming payments could also be robustly bumped via an up to 1kvB
+spend of the incoming deposit output.
+
+### Coinjoins
+
+In the [coinjoin][topic coinjoin] scenario where privacy is the focus but the coinjoin is not
+attempting to be covert, TRUC transactions for the coinjoin itself may be
+worthwhile. The coinjoin may have insufficient feerate for inclusion in the
+blockchain requiring a small bump of a participants’ coins.
+
+Along with TRUC transactions, a P2A output could be added allowing for a
+segregated wallet like a watchtower to pay for the transaction fees alone.
+
+If other participants spend their unconfirmed outputs, TRUC sibling eviction
+will ensure that the other spends can be replaced even if not directly
+conflicting, preserving TRUC topology limits.
+
+Pinning caveat: Participants in the coinjoin may still economically grief the
+transaction by double-spending their own input to the coinjoin, requiring the
+coinjoin to RBF the griefer’s first transaction.
+
+### Lightning Network
+
+Transactions generated in the Lightning Network protocol consist of a few main
+types:
+
+1. Funding transactions: Single-party funded or “dual”-party funded
```

Comment:
```suggestion
1. Funding transactions: 
```

I don't like putting _dual_ in quotes without explaining why (I know it's because many uses of dual-funding are still single funded, just with a different party funding).  Since we don't mention single or dual funding later in the article, I think we can omit them here for simplicity.